tests/ssl-opt.sh

```diff
@@ -695,6 +695,17 @@ run_test() {
     CLI_EXPECT="$3"
     shift 3
 
+    # Expected exit code for the server in response to SIGTERM
+    # Expect 1 for gnutls-serv, otherwise default to expecting 0
```

Comment:
This comment is redundant with the short, readable code below and risks not being kept up to date.